logstash-core/lib/logstash/environment.rb

```diff
@@ -40,8 +40,18 @@ module Environment
             Setting::String.new("http.host", "127.0.0.1"),
             Setting::PortRange.new("http.port", 9600..9700),
             Setting::String.new("http.environment", "production"),
+            Setting::String.new("queue.type", "persisted", true, ["persisted", "memory", "synchronous"]),
```

Comment:
let's push this issue up, technically anything is possible but in terms of usability we have to decide, then we'll adjust here.